bee-tangle/src/urts.rs

```diff
@@ -36,6 +36,8 @@ const MAX_AGE_SECONDS_AFTER_FIRST_CHILD: u8 = 3;
 // used to widen the cone of the tangle.
 const MAX_NUM_CHILDREN: u8 = 2;
 
+const OPTIMAL_NUM_TIPS: u8 = 4;
```

Comment:
Doesn't compile, this needs to be a `usize`.

Reply:
Tnx, replaced u8 to usize, what now?